lib/constants.js

```diff
@@ -0,0 +1,35 @@
+var Constants = {
+  HttpConstants: {
+    /**
+     * Http Verbs
+     *
+     * @const
+     */
+    HttpVerbs: {
+      PUT: 'PUT',
```

Comment:
Small thing... but I would just hardcode these as we do everywhere else in this module... Same for `HttpResponseCodes`..

Take a look at the queue and table files... Keeping things simple has some value :)